articles/hdinsight/hbase/apache-hbase-migrate-new-version.md

```diff
@@ -206,6 +206,22 @@ sudo -u hbase hdfs dfs -Dfs.azure.page.blob.dir="/hbase-wals" -cp <source-contai
    
 1. If the destination cluster is satisfactory, delete the source cluster.
 
+## Migrate HBase using scripts
+
+1. Execute the script [migrate-hbase-source.sh](https://github.com/Azure/hbase-utils/blob/master/scripts/migrate-hbase-source.sh) on the source cluster and [migrate-hbase-dest.sh](https://github.com/Azure/hbase-utils/blob/master/scripts/migrate-hbase-dest.sh) on the destination cluster. Use the following instructions to execute these scripts.
+   > [!NOTE]  
+   > These scripts don't copy the HBase old WALs as part of the migration; therefore, the scripts are not to be used on clusters that have either HBase Backup or Replication feature enabled.
+
+2. On source cluster
+   ```bash
+   sudo bash migrate-hbase-source.sh
+
+3. On destination cluster
+   ```bash
+   sudo bash migrate-hbase-dest.sh  -f <src_default_Fs>
+
+Mandatory argument for the above command:
+
```

Comment:
```suggestion
Mandatory argument for the above command:

   ```
   -f, --src-fs
   The fs.defaultFS of the source cluster
   For example:
   -f wasb://anynamehbase0316encoder-2021-03-17t01-07-55-935z@anynamehbase0hdistorage.blob.core.windows.net
   ```
```